models/issues/stopwatch.go

```diff
@@ -46,6 +47,7 @@ func (err ErrIssueStopwatchAlreadyExist) Unwrap() error {
 type Stopwatch struct {
 	ID          int64              `xorm:"pk autoincr"`
 	IssueID     int64              `xorm:"INDEX"`
+	Issue       *Issue             `xorm:"-"`
```

Comment:
We should probably add a comment such as `// not loaded by default, only when the loading function says it loads it`.
And then a comment for `HasUserStopwatch` such as `// Loads the stopwatches issue`.

Reply:
Rather than go through that I'd rather just remove the issue holder from stopwatch.